x/axelarnet/keeper/msg_server.go

```diff
@@ -68,9 +68,11 @@ func (s msgServer) Link(c context.Context, req *types.LinkRequest) (*types.LinkR
 		sdk.NewEvent(
 			types.EventTypeLink,
 			sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
+			sdk.NewAttribute(types.AttributeKeySourceChain, exported.Axelarnet.Name),
```

Comment:
Source chain could be another Cosmos chain though? We don't know at this point in time. But I guess we can add this for consistency.

Reply:
as of right now, in this module the source chain is always axelarnet: 

```
	if err := s.nexus.LinkAddresses(ctx,
		nexus.CrossChainAddress{Chain: exported.Axelarnet, Address: depositAddress.String()},
		recipient,
	); err != nil {
		return nil, fmt.Errorf("could not link addresses: %s", err.Error())
	}
```